src/ceph_mds.cc

```diff
@@ -108,7 +108,6 @@ int main(int argc, const char **argv)
     }
     else if (ceph_argparse_flag(args, i, "--help", "-h", (char*)NULL)) {
       usage();
-      break;
```

Comment:
It's not obvious when reading that usage() will ultimately call exit(), so a comment would be nice here